### DISCUSSION TITLE
Data filtering not working in DBeaver 21.1.0.202106090921

### DISCUSSION THREAD

**Discussion by artgoldberg** [dbeaver/dbeaver#13273] [Q&A] [answered]
#### System information: 
- Operating system: MacOS Big Sur: 11.2.3 (20D91)
- DBeaver version: 21.1.0.202106090921
- Additional extensions: none

#### Connection specification:
- Database name and version: Microsoft SQL Server 2019 (RTM) - 15.0.2000.5 (X64)
Sep 24 2019 13:48:23
Enterprise Edition: Core-based Licensing (64-bit) on Windows Server 2019 Standard 10.0 (Build 17763: ) (Hypervisor)
- Driver name: don't know
- Do you use tunnels or proxies (SSH, SOCKS, etc)?: Yes, F5 Big-Ip

#### Describe the problem you're observing:
Data filtering does not filter. It does nothing. See this [video](https://www.youtube.com/watch?v=3CnxROzsn-U).

#### Steps to reproduce, if exist:
Display some data and try to filter by value

**Comment by p91paul** [comment, marked as answer]
Try to enlarge that modal. Looks like you are missing the "apply filter" button.

**Comment by artgoldberg** [comment]
Thanks @p91paul , that works. I have two suggestions: Add a feature request which prevents the modal from not showing buttons. In the meantime, add a screenshot of this problem and your suggestion to the DBeaver docs.
Thanks Again
